packages/documentation/src/stories/getting-started/angular/angular.docs.mdx

```diff
@@ -0,0 +1,77 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as AngularStories from './angular.stories';
+import SampleIconsConfig from './sample-icons-config.sample.json?raw';
+import SampleWcAngularManually from './sample-web-components-angular-manually.component.ts?raw';
+
+<Meta of={AngularStories} />
+
+# Angular
+
+The Swiss Post Design System is compatible with Angular. Here are some tips, best practices and information to include components the easiest way.
+
+## Usage of Web Components
+
+In the design-system team, we believe that the [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_Components) technologies should be the source for almost if not all future components.
+
+Currently, we are providing some of our packages with those technologies:
+* [Internet-header](?path=/docs/cfa21cbc-5f8d-4a42-8165-7f6ed20bf133--docs)
+* [Components](/?path=/docs/edfb619b-fda1-4570-bf25-20830303d483--docs)
```

Comment:
We could reuse our package tiles from the home page . This would lighten up the page a bit.
What do you think?

![image](https://github.com/swisspost/design-system/assets/9716662/3b510895-8d19-4d67-9937-e4cbf097bf14)